examples/iterables/test/iterables_test.dart

```diff
@@ -56,6 +56,51 @@ void main() {
       expect(element, '123456');
     });
 
+    test('firstWhere_long_example', () {
+      // #docregion first-where-long
+      bool predicate(String element) {
+        return element.length > 5;
+      }
+
+      void main() {
+        const items = ['Salad', 'Popcorn', 'Toast', 'Lasagne'];
+
+        // You can find with a simple expression:
+        var element1 = items.firstWhere((element) => element.length > 5);
```

Comment:
s/element/item/ because the group is `items`?

Reply:
Good suggestion, thanks. Makes it better match similar, lengthier examples. How does my most recent commit look?